scripts/integration_shards/main.go

```
@@ -0,0 +1,751 @@
+}
+
+var skipLocationPattern = regexp.MustCompile(`^[^:[:space:]]+\.go:[0-9]+:\s*`)
+
+func main() {
+	if err := run(os.Args[1:], os.Stdin, os.Stdout); err != nil {
+		fmt.Fprintf(os.Stderr, "integration shard contract: %v\n", err)
+		os.Exit(1)
+	}
+}
+
+func run(args []string, stdin io.Reader, stdout io.Writer) error {
```

golangci-lint (2.12.2)

[error] cognitive complexity 34 of func `run` is high (> 30)

(gocognit)